app/controllers/jobseekers/job_applications_controller.rb

```diff
@@ -167,4 +168,17 @@ def vacancy
                    job_application.vacancy
                  end
   end
+
+  def send_dfe_analytics_event
+    fail_safe do
+      event = DfE::Analytics::Event.new
+        .with_type(:vacancy_apply_clicked)
+        .with_request_details(request)
+        .with_response_details(response)
+        .with_user(current_jobseeker)
+        .with_data(vacancy_id: StringAnonymiser.new(vacancy.id))
```

Comment:
Not really sure if `vacancy_id` should be anonymised - Only need to do this if it is PII.